src/Main.java

```diff
@@ -1,5 +1,50 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        System.out.println("Задача 1.");
+        int[] arr1 = {10000, 10500, 12300, 23400, 5000};
+        int sum = 0;
+        for (int index = 0; index < arr1.length; index++) {
+            sum += arr1[index];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+        System.out.println("Задача 2.");
+        int[] arr2 = {10000, 10500, 12300, 23400, 5000};
+        int max = -1;
+        for (final int current : arr2) {
+            if (current > max) {
+                max = current;
+            }
+        }
+        int min = 999999;
```

Comment:
Аналогично, если минимальное значение в массиве станет 2000000 (а это возможно), код перестанет работать
```suggestion
        int min = arr2[0];
```